### PR TITLE
Filter with samples unit tests

### PR DESCRIPTION
**Pull request by LCYC1**

### CODE REVIEW

File: tests/test_data_filter_basic.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+from src.fairreckitlib.data.filter.categorical_filter import CategoricalFilter
+from src.fairreckitlib.data.filter.count_filter import CountFilter
+
+from src.fairreckitlib.data.set.dataset_registry import DataRegistry
+
+dataset = DataRegistry('tests/datasets').get_set('ML-100K-Sample')
+filter_kwargs = {'dataset': dataset, 'matrix_name': 'user-movie-rating'}
+
+class TestFilterAge:
+    """Creates a filter object and a dummy data frame to test age filter."""
```
Comment (sivd) [inline tests/test_data_filter_basic.py]: needs to be imperative, remove the s :^)

File: tests/test_data_filter_basic.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+    def test_run_no_country_col(self):
+        """Test a given dataframe with no country column."""
+        df_given = DataFrame({"id": [1, 2, 3, 4, 5], "play_count": [24, 0, -1, 45, 102]})
+        df_result = self.filter_obj.filter(df_given, 'country', ['Sweden'])
+        assert_frame_equal(df_result, self.filter_obj.__empty_df__(df_given))
+
+
+class TestFilterCount:
+    """Creates a filter object and a dummy data frame to test count filter."""
```
Comment (sivd) [inline tests/test_data_filter_basic.py]: also needs to be imperative, plx remove s :^)

**Review by sivd — CHANGES_REQUESTED** [review]
2 comments need to be changed, rest looks good

File: tests/test_data_filter_basic.py (hunk 2 above)
Comment (LCYC1) [reply tests/test_data_filter_basic.py]: a yes ty

**Review by sivd — APPROVED** [review]
awesome

**Comment by LCYC1** [comment]
zo snel big thanks

**Pull request merged**